pom.xml

```diff
@@ -86,7 +86,7 @@
         <maven.source.version>3.0.1</maven.source.version>
         <maven.templating.version>1.0.0</maven.templating.version>
         <mockito.version>2.19.0</mockito.version>
-        <powsybl.core.version>2.5.0</powsybl.core.version>
+        <powsybl.core.version>2.5.2</powsybl.core.version>
         <powsybl.hades2.version>2.5.0</powsybl.hades2.version>
```

Comment:
You must also increase this dependency to 2.5.2 (and create an equivalent of this PR in farao-gse)

Reply:
OK, done for farao-gse. see here:
https://github.com/farao-community/farao-gse/pull/24

Reply:
change:
<powsybl.hades2.version>2.5.0</powsybl.hades2.version>
into 
<powsybl.hades2.version>2.5.2</powsybl.hades2.version>

Reply:
OK, see commit c930173